src/test/java/uk/gov/companieshouse/api/accounts/controller/ProfitAndLossControllerTest.java

```diff
@@ -223,6 +236,8 @@ void updateProfitAndLossCurrentPeriodSuccess() throws DataException {
     @DisplayName("Tests the successful update of a profit and loss resource for previous period")
     void updateProfitAndLossPreviousPeriodSuccess() throws DataException {
 
+        AccountingPeriod period = AccountingPeriod.PREVIOUS_PERIOD;
```

Comment:
There's already an `accountingPeriod` variable declared below. Should we remove this and use that instead, or vice versa?